README.md

```diff
@@ -12,6 +12,12 @@ This style guide is a compilation of [atomic components](http://bradfrost.com/bl
 - Link to the production files at `styleguide/public/assets/`
 
 ## To begin working:
+**Environment setup (mac)**
+
+ - Have [`homebrew`](https://brew.sh/) installed
+ - `brew install nvm`
```

Comment:
Running `echo "source $(brew --prefix nvm)/nvm.sh" >> ~/.profile` from the command line after install seems like the most straightforward way to get things working. That make sense @rpama?